src/components/LanguageBar.js

```diff
@@ -5,11 +5,12 @@ import Tabs from '@material-ui/core/Tabs';
 import Tab from '@material-ui/core/Tab';
 
 
-export default function LanguageBar({supportedLanguages, inputLanguage, outputLanguage, handleInputLanguageChange, handleOutputLanguageChange, classes}) {
+export default function LanguageBar({supportedLanguages, inputLanguage, outputLanguage, languageDetected, handleInputLanguageChange, handleOutputLanguageChange, classes}) {
 
     // Get input and output language codes
     const inputLangCodes = []
     const outputLangCodes = []
+    inputLangCodes.push("auto");
```

Comment:
Since this is a "feature" provided by the backend, should it be returned when asking for all supported languages on startup?

Reply:
I'll add a TODO for that